Automattic/wp-calypso
# Fix/e2e tests block inserted events

### Proposed Changes

This PR fixes broken tests for editor-tracking > block-inserted-events. 

Note: the last test is just being skipped. I've added a comment in the file explaining why, and would plan to return to that test based on what we decide to do. 

### Testing Instructions

1) Set up your testing environment following the instructions here: https://github.com/Automattic/wp-calypso/tree/trunk/test/e2e

2) One set up, run the relevant tests with `yarn jest specs/editor-tracking/editor-tracking__block-inserted-events`. You should see 1 of 1 test suites pass, 25 of 26 tests pass, and 1 of 26 tests skipped.
